geode-core/src/distributedTest/java/org/apache/geode/cache/client/internal/SSLDualServerNoClientAuthDUnitTest.java

```diff
@@ -0,0 +1,213 @@
+import static org.apache.geode.distributed.ConfigurationProperties.SSL_TRUSTSTORE;
+import static org.apache.geode.distributed.ConfigurationProperties.SSL_TRUSTSTORE_PASSWORD;
+import static org.apache.geode.test.dunit.VM.getVM;
+import static org.apache.geode.test.util.ResourceUtils.createTempFileFromResource;
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Properties;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.cache.Cache;
+import org.apache.geode.cache.CacheFactory;
+import org.apache.geode.cache.Region;
+import org.apache.geode.cache.RegionFactory;
+import org.apache.geode.cache.RegionShortcut;
+import org.apache.geode.distributed.Locator;
+import org.apache.geode.test.dunit.VM;
+import org.apache.geode.test.dunit.internal.JUnit4DistributedTestCase;
+import org.apache.geode.test.junit.categories.MembershipTest;
+
+@Category(MembershipTest.class)
+public class SSLDualServerNoClientAuthDUnitTest extends JUnit4DistributedTestCase {
+
+  private Cache cache;
+  private int cacheServerPort;
+  private String hostName;
+
+  private static final String SERVER_1_KEYSTORE = "geodeserver1.keystore";
+  private static final String SERVER_1_TRUSTSTORE = "geodeserver1.truststore";
+
+  private static final String SERVER_2_KEYSTORE = "geodeserver2.keystore";
+  private static final String SERVER_2_TRUSTSTORE = "geodeserver2.truststore";
+
+
+  private static SSLDualServerNoClientAuthDUnitTest instance =
+      new SSLDualServerNoClientAuthDUnitTest();
+
+  @Before
+  public void setUp() {
+    disconnectAllFromDS();
+  }
+
+  @After
+  public void tearDown() {
+    VM serverVM = getVM(1);
+    VM server2VM = getVM(2);
+    VM locator = getVM(3);
+
+    locator.invoke(() -> closeLocatorTask());
+    server2VM.invoke(() -> closeCacheTask());
+    serverVM.invoke(() -> closeCacheTask());
+  }
+
+  @Test
+  public void testSSLServerWithNoAuth() {
+    VM serverVM = getVM(1);
+    VM server2VM = getVM(2);
+
+    VM locator = getVM(3);
+
+    Integer locatorPort = locator.invoke(() -> {
+      return setUpLocatorTask();
+    });
+    boolean cacheServerSslenabled = true;
+
+    serverVM.invoke(() -> setUpServerVMTask(locatorPort));
+    server2VM.invoke(() -> setUpServerVMTask(locatorPort));
+
+    server2VM.invoke(() -> doServerRegionTestTask());
+    serverVM.invoke(() -> doServerRegionTestTask());
+  }
+
+  private void createCache(Properties props) throws Exception {
+    cache = new CacheFactory(props).create();
+    if (cache == null) {
+      throw new Exception("CacheFactory.create() returned null ");
+    }
+  }
+
+  private Integer setUpLocator() throws Exception {
+    Properties gemFireProps = new Properties();
+
+    String cacheServerSslprotocols = "any";
```

Comment:
Could be final - I saw some other similarly un-mutated strings declared final so we should at least try and be consistent.